Automattic/wp-calypso
# Untangling: Add Advanced Tools -> Deployments

Fixes:

- https://github.com/Automattic/dotcom-forge/issues/9617

## Proposed Changes

This PR adds the Advanced Tools -> Deployments subtab under the `untangling/hosting-menu` flag.

<img width="949" alt="image" src="https://github.com/user-attachments/assets/8d905345-aacd-4107-865b-bbe6a2208262">

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click an Atomic site
3. Go to Advanced Tools -> Deployments subtab.
4. Verify that this tool is working (try connecting to a repo etc)
5. Regression testing:
    - Go to /sites?flags=-untangling/hosting-menu
    - Click an Atomic site
    - Verify that the original Deployments tab is still working.

## Pre-merge Checklist
